functions/src/utils.ts

```diff
@@ -15,18 +15,25 @@ export const logMemory = () => {
   }
 }
 
-export const mapAsync = async <T, U>(
```

Comment:
I thought this was a cool function!

Reply:
A function has to be useful to be cool! It makes sense to me but not if I'm not using it.